### ISSUE TITLE
Real-Data in your paper

### ISSUE THREAD

**Issue by Gavin-Evans** [AliaksandrSiarohin/pose-gan#51]
Hello! Professor
Could you explain the Real-Data mentioned in the paper? Because you did not give a description about Real-Data in the paper.
Email：xrinozpg616@gmail.com
thank you.

**Comment by AliaksandrSiarohin** [comment]
Hello,
I don't really get what you mean. Can you please elaborate? You want description of the datasets, or descruption of datasets preprocessing? Or how to download the data?

**Comment by Gavin-Evans** [comment]
I'm sorry!
In your paper:Appearance and Pose-Conditioned Human Image Generation using Deformable GANs
You mentioned the Real-Data evaluation indicators in Table 1，However you did not describe how Real-Data is calculated and what it mean, so I am very confused

**Comment by AliaksandrSiarohin** [comment]
Yes so Real data is just Inception score and detection scores computed in the test part of our dataset, e. g. Real data. So this serves as an upper bound to the scores that you can possibly acchive. For ssim it is 1, because it is the maximum, basically it is ssim(gt, gt), where gt is a ground truth.

**Comment by Gavin-Evans** [comment]
I got it, thank you
